doc/update-gh-pages.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# Deploy repository documentation
+
+# Variables
+bold=$(tput bold)
+normal=$(tput sgr0)
+script_path="$( cd "$(dirname "$0")" ; pwd -P )"
+project_path=${script_path}/..
+clone_folder=/tmp/update-repos
+
+repository_name="ping-python"
+repository_githash=$(git -C ${project_path} rev-parse HEAD)
+
+# Functions
+echob() {
+    echo "${bold}${1}${normal}"
+}
+
+echob "Check build type."
+# Do not build pull requests
+if [ ${TRAVIS_PULL_REQUEST} != "false" ]; then
+    echo "- Do not deploy PRs."
+    exit 0
+fi
+
+echob "Check branch."
+# Do only build master branch
+if [ ${TRAVIS_BRANCH} != "master" ]; then
+    echo "- Only master branch will be deployed."
+    exit 0
+fi
+
+echob "Check git configuration."
+if [ "${TRAVIS}" = "true" ] || ! git config --list | grep -q "user.name"; then
+    # Config for auto-building
+    echo "- Git configuration does not exist, a new one will be configured."
+    git config --global user.email "support@bluerobotics.com"
+    git config --global user.name "BlueRobotics-CI"
+else
+    echo "- Git configuration already exist."
+fi
+
+echob "Build doxygen documentation."
+if ! ( cd $script_path && doxygen "Doxyfile" ); then
+    echo "- Doxygen generation failed."
+    exit 1
+fi
+echo "- Check files"
+ls "${script_path}/html/"
+
+repo_path=${clone_folder}/${repository_name}
+echo "- Clone ${repository_name}"
+rm -rf ${repo_path}
+git clone https://${GITHUB_TOKEN}@github.com/bluerobotics/${repository_name} ${repo_path}
```

Comment:
Have you configured the token ?

Reply:
I believe it is configured the same for ping-arduino and nothing needs to be done here. The token is linked to BR travis account.

Reply:
I think the token is used for each repository individually.
![image](https://user-images.githubusercontent.com/1215497/51856395-8430d400-2327-11e9-8008-dc19f89317ee.png)
![image](https://user-images.githubusercontent.com/1215497/51856412-8d21a580-2327-11e9-9825-f2054090802b.png)

Reply:
it's already configured for ping-arduino